### PR TITLE
[Feat] Add User Management Page (Front-end)

### PR DESCRIPTION
**Pull request by 3nvi**
## Background

Why are you making this change? Please provide a reference to any related issues and PRs
This PR adds user management back to the app

## Changes

- Add user management page
- Allow `admin` invitatinos
- Allow user listing

## Testing

- How did you test your change?

Not proper testing until related lambdas are implemented

### CODE REVIEW

**Comment by 3nvi** [comment]
@austinbyers Let me know when you have added the related lambdas. Specifically we need:

* `listUsers`
* `inviteUser`

**Comment by austinbyers** [comment]
@3nvi we already have `InviteUser` (see https://github.com/panther-labs/panther/blob/master/api/lambda/users/models/api.go)

So is list users all that is missing? Presumably also deleting users?

**Comment by 3nvi** [comment]
> So is list users all that is missing? Presumably also deleting users?

Correct

**Comment by 3nvi** [comment]
@austinbyers Apparently, the delete user functionality [was never exposed](https://github.com/panther-labs/users-api/tree/master/api). That's why I didn't/forgot-to add it

**Comment by 3nvi** [comment]
> So is list users all that is missing? Presumably also deleting users?

Forgot to add the delete user resolver. Added it

File: api/graphql/schema.graphql [hunk 1]
```diff
@@ -650,6 +653,23 @@ type User {
...
+type InviteUserResponse {
+  id: ID!
+}
+
+input InviteUserInput {
+  givenName: String
+  familyName: String
+  email: AWSEmail
+  phoneNumber: AWSPhone
+  role: RoleNameEnum
```
Comment (austinbyers) [inline api/graphql/schema.graphql]: FYI I removed this field in #100

Comment (3nvi) [reply api/graphql/schema.graphql]: It's safely ignored by the lambda. Helps to have it here.

**Pull request merged**